# Same as a regular imu, but without the magnetic field
Header header

geometry_msgs/Vector3 angular_velocity
geometry_msgs/Vector3 linear_acceleration

float32 roll
float64 pitch
